sys-apps/hdparm/hdparm-9.65-r2.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs flag-o-matic
+
+DESCRIPTION="Utility to change hard drive performance parameters"
+HOMEPAGE="https://sourceforge.net/projects/hdparm/"
+SRC_URI="mirror://sourceforge/hdparm/${P}.tar.gz"
+
+# GPL-2 only
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="static"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-9.60-build.patch
+	"${FILESDIR}"/${P}-graceful-lfs.patch
+)
+
+src_prepare() {
+	default
+
+	use static && append-ldflags -static
+}
+
+src_configure() {
+	append-lfs-flags
```

Comment:
Could you send a patch upstream please to do `AC_SYS_LARGEFILE` in configure.ac?

Reply:
Hi Sam, do you mean hdparm upstream?
I don't think there is configure.ac in hdparm source code - it has only plain Makefile
Also hdparm is distributed through sourceforge, and patch submission for sourceforge projects is not as straight forward AFAIU.

Reply:
@thesamesam Hi Sam, can you please provide clarifications for my questions above?

Reply:
> Hi Sam, do you mean hdparm upstream? I don't think there is configure.ac in hdparm source code - it has only plain Makefile Also hdparm is distributed through sourceforge, and patch submission for sourceforge projects is not as straight forward AFAIU.

Right, hdparm upstream. But I'd missed that it's not autoconf.

Let's just go with this then.